text_reading/docker/docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
         image: allenai/scienceparse:2.0.3
         ports:
             - "8080:8080"
-    quantities:
+    grobid:
```

Comment:
i wonder if we should name it `grobidquantities` instead, because grobid is a pdf parser that we're not using